drivers/sensor/bme680/Kconfig

```diff
@@ -0,0 +1,166 @@
+# Kconfig.bme680 - BME680 temperature/pressure/gas sensor configuration options
+
+#
+# Copyright (c) 2018 Laird
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+menuconfig BME680
+	bool "BME680 sensor"
+	depends on I2C
+	default	n
+	help
+	 Enable driver for BME680 I2C-based temperature, humidity,
+	 gas and pressure sensor.
+
+if BME680
+
+config BME680_DEV_NAME
```

Comment:
Device name, i2c address, and i2c device should come from device tree instead of kconfig.